pandas/core/dtypes/common.py

```diff
@@ -90,9 +90,11 @@ def ensure_categorical(arr):
 def ensure_int64_or_float64(arr, copy=False):
```

Comment:
can you type this (ArrayLike) -> np.array

Reply:
done